packages/amqp/vitest.config.mts

```diff
@@ -12,13 +12,21 @@ export default defineConfig({
     environment: 'node',
     reporters: ['default'],
     coverage: {
+      provider: 'v8',
       include: ['lib/**/*.ts'],
-      exclude: ['lib/**/*.spec.ts', 'lib/**/*.test.ts', 'test/**/*.*', 'lib/types/**/*.*'],
+      exclude: [
+        'lib/**/*.spec.ts',
+        'lib/**/*.test.ts',
+        'test/**/*.*',
+        'lib/types/**/*.*',
+        'lib/AmqpExchangePublisherManager.ts',
```

Comment:
it's problematic to write tests for this functionality, as we don't yet have any logic in place for autocreating exchanges and autosubscribing queues. this will be done next.